### DISCUSSION TITLE
Visualización Web de Proyectos

### DISCUSSION THREAD

**Discussion by JoseLuisGalvis** [community/community#23447] [New to GitHub]
<p>Good Morning to All: :coffee:</p>
<p>I uploaded the Project titled “Project-4to” at the address :desktop_computer: <a href="https://joseluisgalvis.github.io/Proyecto-4to/" class="inline-onebox" rel="noopener nofollow ugc">Pole Position - Inicio</a>, as part of the repository <a href="https://github.com/JoseLuisGalvis/Proyecto-4to.git" class="inline-onebox" rel="noopener nofollow ugc">GitHub - JoseLuisGalvis/Proyecto-4to: Trabajo Practico Obligatorio Codo a Codo</a>.</p>
<p>It’s status is Ok, :ballot_box_with_check:, but the issue is that it does not read the css stylesheet. :no_entry_sign:</p>
<p>Please, Help me to solve it. :sos: :sos: :sos:</p>
<p>Thank you all very much! :wave: :wave: :wave:</p>
<p>Greetings…</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/d/cd8723c6f1c1d0388d02fb6ff7c5d32534d8011b.png" data-download-href="/uploads/short-url/tkbpR43ikrABY9Kxs3NiNGbohAL.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/92390795/181077135-66d96e31-f666-4aef-bb30-b364fdcf2d33.png" alt="image" data-base62-sha1="tkbpR43ikrABY9Kxs3NiNGbohAL" width="690" height="496" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/d/cd8723c6f1c1d0388d02fb6ff7c5d32534d8011b_2_690x496.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/d/cd8723c6f1c1d0388d02fb6ff7c5d32534d8011b_2_1035x744.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/c/d/cd8723c6f1c1d0388d02fb6ff7c5d32534d8011b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/d/cd8723c6f1c1d0388d02fb6ff7c5d32534d8011b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1093×787 63.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>first is that, you’re loading from a local resource, which is not allowed, try to get all the URLs of those files, either relative or absolute, and just make sure you know they are the correct URLs</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/c/7c39e67bf3b165b17dd1a6ff4798dd64fcd2c1b0.png" data-download-href="/uploads/short-url/hIXfWKhFBEjpUbggonAGbyJ5eX6.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077138-085ce757-83d1-487c-b3a8-f33e68bb8bd3.png" alt="image" data-base62-sha1="hIXfWKhFBEjpUbggonAGbyJ5eX6" width="334" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/c/7c39e67bf3b165b17dd1a6ff4798dd64fcd2c1b0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">466×697 42.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/7/3746d7443afb9d7ff2f7b1e7359e3c5eba265977.png" data-download-href="/uploads/short-url/7T02hlrM1qsw3shDCmv0t02dW6z.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077141-42bcfb24-5518-40b1-9906-d0a0934f377f.png" alt="image" data-base62-sha1="7T02hlrM1qsw3shDCmv0t02dW6z" width="690" height="85" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/7/3746d7443afb9d7ff2f7b1e7359e3c5eba265977_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">897×111 6.21 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>as you can see, you’re loading it from your computer</p>
<p>the other one is 404s</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181077142-5eb32872-4d58-4b00-b36c-a3d700f7a6ac.png" alt="image" data-base62-sha1="t3stpS7TToSr97QoI1SJunXHmnS" width="331" height="265"></p>
<p>for example, the correct URL of one that you load is</p>
<p><code>https://joseluisgalvis.github.io/Proyecto-4to/2css/pp1.css</code></p>
<p>but at your source, it is</p>
<p><code>https://joseluisgalvis.github.io/C22067_DNI96065434/2css/pp1.css</code></p>

**Comment by JoseLuisGalvis** [comment, marked as answer]
<p>Hi Mr <a class="mention" href="/jdevstatic">@jdevstatic</a> :</p>
<p>Thank you very much for your help! :sos: :sos: :sos:<br>
I will follow your instructions and will let you know the result as soon as possible. :phone:<br>
Thank you very much for your time, :hourglass_flowing_sand:<br>
willingness to help me :ballot_box_with_check:<br>
and valuable guidance, :+1:<br>
Best regards!</p>

**Comment by JoseLuisGalvis** [comment]
<p>Good Evening  Mr. <a class="mention" href="/jdevstatic">@jdevstatic</a>:<br>
:argentina:</p>
<p>21:21 hrs. <strong>100% Online Project</strong>, Thanks to its proper orientation, :beers:</p>
<p>Thanks Github Community!<br>
<img src="https://user-images.githubusercontent.com/92390795/181077143-079849f3-dd42-473a-8b4e-cde8e9fcfd4b.png" alt="image" data-base62-sha1="8Mnm9SCJ7XGApC420RXoKWQKq9M" width="150" height="84"></p>
<p>Best regards!</p>
<details>
<summary>
Summary</summary>
<div class="spoiler">
<p><strong>This text will be hidden</strong></p>
</div>
</details>
